modules/distr-tracing-config-otel-collector.adoc

```diff
@@ -1,39 +1,126 @@
+.sample OpenTelemetry collector custom resource file
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: tracing-system
+spec:
+  mode: deployment
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+    exporters:
+      jaeger:
+        endpoint: jaeger-production-collector-headless.tracing-system.svc:14250
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: []
+          exporters: [jaeger]
+----
+
+[NOTE]
+====
+If a component is configured, but not defined within the `service` section then it is not enabled.
+====
 
 .Parameters used by the Operator to define the OpenTelemetry Collector
 [options="header"]
-[cols="l, a, a"]
+[cols="l, a, a, a"]
 |===
-|Parameter |Description |Values
+|Parameter |Description |Values |Default
+|receivers:
+|A receiver is how data gets into the Collector. By default, no receivers are configured. There must be at least one enabled receiver for a configuration to be considered valid. Receivers are enabled by being added to a pipeline.
+|`otlp`, `jaeger`
+|None
+
+|receivers:
+  otlp:
+|The `oltp` and `jaeger` receivers come with default settings, specifying the name of the receiver is enough to configure it.
 |
 |
+
+|processors:
+|Processors are run on data between being received and being exported. By default, no processors are enabled.
 |
-|===
+|None
 
+|exporters:
+|An exporter is how you send data to one or more backends/destinations. By default, no exporters are configured. There must be at least one enabled exporter for a configuration to be considered valid. Exporters are enabled by being added to a pipeline. Exporters may come with default settings, but many require configuration to specify at least the destination and security settings.
+|`logging`, `jaeger`
+|None
 
-.Parameters passed to the Collector
-[options="header"]
-[cols="l, a, a"]
-|===
-|Parameter |Description |Values
-|spec:
- collector:
-  options: {}
+|exporters:
+ jaeger:
+  endpoint:
+
+|The `jaeger` exporter’s endpoint must be of the form `<name>-collector-headless.<namespace>.svc`, with the name and namespace of the Jaeger deployment, for a secure connection to be established.
+|
+|
+
+|exporters:
+ jaeger:
+  tls:
+   ca_file:
+|Path to the CA certificate. For a client this verifies the server certificate. For a server this verifies client certificates. If empty uses system root CA.
+|
 |
+
+|service:
+  pipelines:
+|Components are enabled by adding them to a pipeline under `services.pipeline`.
+|
+|
+
+|service:
+  pipelines:
+    traces:
+      receivers:
+|You enable receivers for tracing by adding them under `service.pipelines.traces`.
+|
+|None
+
```

Comment:
Probably should have a row in the table for service.pipelines.traces.processors here, since any processors need to be listed there as well as in the processors section above.